front/src/applications/stdcm/hooks/useStdcm.ts

```diff
@@ -155,6 +155,16 @@ const useStdcm = ({
     payload: PostTimetableByIdStdcmApiArg
   ) => {
     try {
+      const currentSimulation = await createSimulation(
+        currentSimulationInputs,
+        payload,
+        response,
+        undefined
+      );
```

Comment:
No need to pass undefined as the param is optional
```suggestion
    try {
      const currentSimulation = await createSimulation(currentSimulationInputs, payload, response);
```